src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/System.Text.RegularExpressions.Tests.csproj

```diff
@@ -72,7 +73,7 @@
     <Compile Include="$(CommonTestPath)System\Diagnostics\DebuggerAttributes.cs" Link="Common\System\Diagnostics\DebuggerAttributes.cs" />
 
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="$(MicrosoftCodeAnalysisVersion)" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.XUnit" Version="$(CompilerPlatformTestingVersion)" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeFix.Testing" Version="$(CompilerPlatformTestingVersion)" />
```

Comment:
I assume you changed this as the package brings xunit.assert in transitively? What if we would make sure that Microsoft.DotNet.XUnitAssert's assembly identity matches xunit.assert's one. Would we then be able to consume these packages and exclude its transitives?

Reply:
Yup, this is the approach I took in https://github.com/dotnet/arcade/pull/13510. If that sounds good to you, I will take that change and then retry this PR on top of that. A whole lot of problems will go away.